cypress/support/commands/rancher-api-commands.ts

```diff
@@ -14,15 +14,18 @@ Cypress.Commands.add('login', (
   username = Cypress.env('username'),
   password = Cypress.env('password'),
   cacheSession = true,
+  skipNavigation = false,
 ) => {
   const login = () => {
     cy.intercept('POST', '/v3-public/localProviders/local*').as('loginReq');
 
-    LoginPagePo.goTo(); // Needs to happen before the page element is created/located
+    if (!skipNavigation) {
+      LoginPagePo.goTo(); // Needs to happen before the page element is created/located
+    }
     const loginPage = new LoginPagePo();
 
     loginPage
-      .checkIsCurrentPage();
+      .checkIsCurrentPage(!skipNavigation);
```

Comment:
When we would logout we'd have a query parameter `?logged-out` which would cause the exact match to fail. I added this but maintained the default behavior. Maybe we'd want another argument in the future but for now the only time we want this to occur is if we skip navigation.